LiteCore/tests/DataFileTest.cc

```diff
@@ -416,6 +419,41 @@ N_WAY_TEST_CASE_METHOD (DataFileTestFixture, "DataFile Compact", "[DataFile]") {
     db->compact();
 }
 
+TEST_CASE("CanonicalPath") {
+#ifdef _MSC_VER
+    const char* startPath = "C:\\folder\\..\\subfolder\\";
+    string endPath = "C:\\subfolder\\";
+#else
+    auto tmpPath = FilePath::tempDirectory().path();
+    auto startPath = tmpPath + "folder/";
+    ::mkdir(startPath.c_str(), 777);
+    startPath += "../subfolder/";
+    auto endPath = tmpPath + "subfolder";
+    ::mkdir(endPath.c_str(), 777);
+#if __APPLE__
+    endPath = "/private" + endPath;
```

Comment:
I don't believe this is true on iOS: tempDirectory() will be a path somewhere inside the app's data directory, not inside `/tmp`.